docs/reference/mapping/runtime.asciidoc

```diff
@@ -42,6 +42,28 @@ front, and can use runtime fields to amend the mapping at any time. Using
 runtime fields allows for a smaller index and faster ingest time, which
 combined use less resources and reduce your operating costs.
 
+[discrete]
+[[runtime-performance]]
+=== Performance
+You can use <<script-fields,script fields>> to access values in `_source` and
+return calculated values based on a script valuation. Runtime fields have these
+same capabilities, but provide greater flexibility because you can query and
+aggregate on runtime fields in a search request. Script fields can only fetch
+values.
+
+Similarly, you could write a <<query-dsl-script-query,script query>> that
+filters documents in a search request based on a script. Runtime fields provide
+a very similar feature that is more flexible. You write a script to create
+field values and they are available everywhere, such as
+<<search-fields,`fields`>>, <<query-dsl, all queries>>, and
+<<search-aggregations, aggregations>>.
```

Comment:
Shall we mention also for completeness script based sorting? (https://www.elastic.co/guide/en/elasticsearch/reference/7.13/sort-search-results.html#_script_based_sorting ) . I am tempted to expand this PR to address the why behind runtime fields, what they can replace out of the existing ways scripts can already be used in search, to then say what you are saying here around performance.

Reply:
++ @javanna -- I added script-based sorting in 89cfbdd